build-tools/packages/build-cli/src/instructionalPromptWriter.ts

```diff
@@ -76,11 +76,6 @@ export const ADOPipelineLinks = new Map<ReleasePackage | ReleaseGroup | undefine
 		"@fluid-tools/benchmark",
 		"https://dev.azure.com/fluidframework/internal/_build?definitionId=62",
 	],
-	[
-		"@fluidframework/test-tools",
-		"https://dev.azure.com/fluidframework/internal/_build?definitionId=13",
-	],
-	["tinylicious", "https://dev.azure.com/fluidframework/internal/_build?definitionId=22"],
```

Comment:
This PR makes test-tools now build as part of the client release group so it needs no separate pipeline.

Tinylicious has been like that (build as part of server/routerlicious) for a while, so just cleaning up its entry here.

For 1.x builds we pin to build-tools 0.5.0 or 0.6.0, so these entries will still exist there, which is more in sync with the status of the 1.x builds (test-tools and tinylicious have their own pipelines) (and not sure it matters if these entries are here or not).